examples/dataservice-example/CMakeLists.txt

```
@@ -15,37 +15,41 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
-set(OLP_SDK_DATASERVICE_WRITE_EXAMPLE_TARGET dataservice-write-example)
+set(OLP_SDK_DATASERVICE_EXAMPLE_TARGET dataservice-example)
 set(OLP_SDK_EXAMPLE_SUCCESS_STRING "Publish Successfull")
 set(OLP_SDK_EXAMPLE_FAILURE_STRING "Publish Failed!")
```

```suggestion
set(OLP_SDK_EXAMPLE_FAILURE_STRING "Example is finished with errors!")
```
or something similar...